01-Git_Bash_Make/Bash_o_script.script

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+if [ ! -d ~/tmp ]; then
+	mkdir ~/tmp
+fi
+
+touch ~/tmp/guessanumber
+
+cat guessanumber.c > ~/tmp/guessanumber
```

Comment:
- for copy, please use cp command
- you should copy all c/cpp files /tmp/guesanumber